app/controllers/application_controller.rb

```diff
@@ -1,20 +1,46 @@
-# typed: true
+# typed: false
```

Comment:
this was because sorbet didn't recognize `rescue_from`.  is there a way to tell it "no really this method exists for this class"?

Reply:
Did you try `typed: true`? I've found that works more often than `strict`

Reply:
yeah, but no luck with that either.